src/assets/css/panel.css

```diff
@@ -94,7 +94,8 @@
 }
 
 .panel-bordered-force {
-    overflow: auto;
+    overflow-x: auto;
+    overflow-y: hidden;
```

Comment:
I can't really see any difference in HU editor but it doesn't break anything either so maybe I'm missing something. Thanks

Reply:
The issue happened only on some specific cases, ie. on page 1 there's no scrollbar, but on page 2 there is; then on page 3 there isnt one again. I think it's related to the content of the pages.